tools/syz-declextract/README.md

```diff
@@ -0,0 +1,41 @@
+# syz-declextract
+## Linux Kernel (For testing purposes)
+```
+export KERNEL=$PWD/linux-stable
+git clone \
+https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git $KERNEL
```

Comment:
I would use mainline kernel here:
git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git

I think we will extract descriptions from it.

Reply:
Done